ParkingTicketFrontEnd/Models/ParkingTicket_VM.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace ParkingTicketFrontEnd.Models
+{
+    public class ParkingTicket_VM
+    {
+        public string ZipCode { get; set; }
+        public string VehicleTag { get; set; }
+        public Int32 OffenseCode { get; set; }
+        public string ParkingTicketMessage { get; set; }
+
+        public ParkingTicket_VM()
```

Comment:
you don't need this.

Reply:
i mean the constructor. with your public methods, that'll be just fine.

Reply:
I will remove this in the next push to the branch.